app/src/main/java/com/cornellappdev/android/eatery/components/WaitTimesComponent.java

```diff
@@ -67,6 +75,43 @@ public void onClick(View v) {
         });
         mWaitTimesChart = view.findViewById(R.id.wait_time_chart);
         mWaitTimesChart.setNoDataText("");
+
+        mWaitTimesChart.setOnTouchListener(new View.OnTouchListener() {
+            @Override
+            public boolean onTouch(View v, MotionEvent event) {
+                if (mFingerJustTapped) {
+                    // Measure the yDiff directly after the tap, high y diff means motion was a
+                    // vertical swipe
+                    float yDiff = Math.abs(event.getY() - mLastY);
+                    float xDiff = Math.abs(event.getX() - mLastX);
+                    // If it was not a vertical swipe
+                    if (xDiff > yDiff) {
+                        scrollView.requestDisallowInterceptTouchEvent(true);
+                        mVerticalScrolling = false;
+                    } else {
+                        // This variable (when true) is used below in onValueSelected
+                        // to prevent interaction with the mWaitTimesChart. I couldn't find
+                        // a cleaner way of disabling interaction with the chart
+                        mVerticalScrolling = true;
+                    }
+                    mFingerJustTapped = false;
+                }
+
+                if(event.getAction() == MotionEvent.ACTION_DOWN) {
```

Comment:
space after if

Reply:
might be a good idea to do a `CTRL + ALT + Shift + L` to auto-lint everything